e2e/actors/tabs.js

```diff
@@ -17,6 +17,18 @@ function tabFieldSelector(pathToField) {
   return `${selector}//*[contains(@class,"complex-panel-simple-field") and .//th/span[text()="${fieldName}"]]`;
 }
 
+function tabTagSelector(pathToField) {
+  let path = [].concat(pathToField);
+  let fieldName = path.splice(-1, 1)[0];
+  let selector = TAB_CLASS_SELECTOR;
+
+  path.forEach(step => {
+    selector = `${selector}//*[@class="complex-panel" and .//*[@class="complex-panel-title" and .//*[text()="${step}"]]]`;
+  });
+
+  return `${selector}//*[contains(@class,"govuk-tag") and text()="${fieldName}"]`;
+}
+
```

Comment:
Im a bit nieve to the functionality of this @tomaszpowroznik , could you elaborate on what it does please?

Reply:
This piece of code locates tag inside specified path in tab. Used to check if particular section is or not is tagged
![tag in tab](https://user-images.githubusercontent.com/5861660/126454267-5090cb2b-5661-4e74-b1e6-03a3e43ed2b1.jpg)